src/routes.js

```diff
@@ -1,9 +1,22 @@
 import React from 'react';
-import { Route } from 'react-router';
-import { HelloWord } from 'views/HelloWord';
+import { Switch, Route } from 'react-router-dom';
+import HelloWorld from 'views/HelloWorld';
 
-export default (
-    <Route>
-        <Route path="/" component={HelloWord} />
-    </Route>
-);
+// This is a class-based component because the current
+// version of hot reloading won't hot reload a stateless
+// component at the top-level.
+/* eslint-disable react/prefer-stateless-function */
+
+class App extends React.Component {
+    render() {
+        return (
+            <div>
```

Comment:
I guess this `div` is not necessary

Reply:
hmm, I remember something about this component needs to have this `div` around, but I'm going to check before could say something in definitive :)